indico/modules/events/registration/client/js/form_submission/RegistrationFormSubmission.jsx

```diff
@@ -139,13 +158,16 @@ export default function RegistrationFormSubmission() {
           ))}
           {renderPluginComponents('regformAfterSections')}
           <div>
-            {isManagement && <EmailNotification />}
+            {isManagement && (
+              <>
+                <EmailNotification />
+                <VisibilityOverride />
```

Comment:
I would not make this part of the "modify registration" dialog but rather do it similar to what we have for changing consent when viewing your own registration. Combined with my other comment, this would simply be a "hide from participant list" toggle.